src/rbac/defaultRoleManager.ts

```diff
@@ -90,7 +90,7 @@ export class DefaultRoleManager implements RoleManager {
     }
 
     if (!this.hasRole(name)) {
-      throw new Error('error: name does not exist');
+      return [];
```

Comment:
Why remove the error?

Reply:
I think it is not an error,  so remove this error.

Reply:
I think it is not an error,  so remove this error.